modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+= Production Readiness Checklist 
+
+Before running production workloads on Redpanda it’s worth ensuring that the cluster is deployed and configured according to best practices. The checklist in this guide will help you ensure that you are set up for success. In this guide we identify three levels of production readiness - level 1 through level 3. At level 3 you will be ready to run mission critical workloads in the largest of enterprises, whereas smaller organizations or less critical workloads may suffice at level 1.
+
+Redpanda provides multiple resources for deploying a production cluster. We recommend you utilize the Automated deployment instructions with the Redpanda Ansible collection. If you cannot use this method, we provide manual deployment instructions, which are more involved.
+
+== Checklists
+
+=== Level 1 checklist
```

Comment:
What do we mean by levels? Are those levels considered for anything from support side?

Reply:
Levels imply the completeness of their production state. They want to checkbox everything in all three levels. Think of it from the perspective of production maturity.

Reply:
That's insufficient @hcoyote . We need to clearly describe/define each level. I get the criteria is the actual checklist--satisfy this list and you have achieved Level X readiness, but what, exactly defines the level? For example, why would Level 1 be sufficient for a small or or less mission-critical workloads?
If they are not aligned with support, then all the more reason to clearly state exactly how we define each.

Reply:
See the level definitions submitted by @tmgstevens . I've updated to present as lists rather than sentences with 3-4 checklist items--this way the level definition/summary will easy/quicker for readers to scan/take in.